examples/webgpu_postprocessing_sss.html

```diff
@@ -128,33 +128,54 @@
 				renderer.inspector = new Inspector();
 				document.body.appendChild( renderer.domElement );
 
+				// post-processing
+
 				postProcessing = new THREE.PostProcessing( renderer );
 
-				const scenePass = pass( scene, camera );
-				scenePass.setMRT( mrt( {
-					output: output,
-					velocity: velocity
+				// pre-pass
+
+				const prePass = pass( scene, camera );
+				prePass.name = 'Pre-Pass';
+				prePass.transparent = false;
+
+				prePass.setMRT( mrt( {
+					output: velocity
 				} ) );
 
-				const scenePassColor = scenePass.getTextureNode( 'output' );
-				const scenePassVelocity = scenePass.getTextureNode( 'velocity' );
-				const scenePassDepth = scenePass.getTextureNode( 'depth' );
+				const prePassDepth = prePass.getTextureNode( 'depth' ).toInspector( 'Depth', () => prePass.getLinearDepthNode() );
+				const prePassVelocity = prePass.getTextureNode( 'output' ).toInspector( 'Velocity' );
+
+				// scene pass
+
+				const scenePass = pass( scene, camera ).toInspector( 'Color' );
 
 				// sss
 
-				const sssPass = sss( scenePassDepth, camera, dirLight );
-				sssPass.shadowIntensity.value = 0.3;
+				const sssPass = sss( prePassDepth, camera, dirLight );
 				sssPass.maxDistance.value = 0.2;
 				sssPass.useTemporalFiltering = true;
 
-				// composite
+				// scene context
+
+				const sssSample = sssPass.getTextureNode().sample( screenUV ).r;
+
+				const sssContext = context( {
```

Comment:
For the future, do you think we could simplify this bit somehow? Maybe:
```js
scenePass.contextNode = context( {
    sss: sssPass.getTextureNode().sample( screenUV ).r
} );
```

The instance of `SSSNode` has a reference to the directional main light so maybe you could hide some of this code in `AnalyticLightNode`.